test/e2e/servers/graphql-main-server/galoy-pay-testing.spec.ts

```diff
@@ -30,24 +30,32 @@ import {
   PID,
   startServer,
 } from "test/helpers"
+import { loginFromPhoneAndCode, updateUsername } from "test/helpers/account-creation-e2e"
 
 let apolloClient: ApolloClient<NormalizedCacheObject>,
   disposeClient: () => void = () => null,
   receivingWalletId: WalletId,
   serverPid: PID
-const receivingUsername = "user15"
+const receivingUsername = "user15" as Username
 const receivingUserRef = "G"
 const sendingUserRef = "D"
 
 const { phone, code } = getPhoneAndCodeFromRef(sendingUserRef)
+const { phone: phoneRecipient, code: codeRecipient } =
+  getPhoneAndCodeFromRef(receivingUserRef)
 
 beforeAll(async () => {
   await initializeTestingState(defaultStateConfig())
+
+  serverPid = await startServer("start-main-ci")
+
+  await loginFromPhoneAndCode({ phone, code })
```

Comment:
what is the purpose of this? you are not using the client

Reply:
to create the user/account in the db